tests/unittests/CoreGraphics.Drawing/CGContextDrawing_ShadowTests.cpp

```diff
@@ -50,3 +50,53 @@ DISABLED_DRAW_TEST_F(CGContext, ShadowWithRotatedCTM, WhiteBackgroundTest<>) {
 
     CGContextStrokeRect(context, rect);
 }
+
+DISABLED_DRAW_TEST_F(CGContext, ShadowOverlap, WhiteBackgroundTest<>) {
```

Comment:
these three are disabled for the same reason as the other shadow tests - our platform has a minor divergence in the blurriness and darkness of the shadows compared to the reference platform. (this difference is small enough that it's difficult to distinguish by eye, so it's not a blocker) manually verified that these test results diverged only in that respect.